te3/tutorials/data-security/data-security-setup-rls.md

```diff
@@ -0,0 +1,151 @@
+  - Execute it and observe the results.
+  - Replace `USERNAME()` or `USERPRINCIPALNAME()` in dynamic RLS with a known value from the Security Table.
+  - Re-run the DAX Query and validate that the results appear as expected. Repeat until satisfied.
+
+
+<figure style="padding-top: 15px;">
+  <img class="noscale" src="~/images/data-security/data-security-rls-validation-example.png" alt="Data Security Create Role" style="width: 550px;"/>
+  <figcaption style="font-size: 12px; padding-top: 10px; padding-bottom: 15px; padding-left: 75px; padding-right: 75px; color:#00766e"><strong>Figure 5:</strong> An example of how RLS can be validated from the DAX query window by using the Filter Expression inside of an Iterator over the Table (or part of the table, like the user alias). In this example, the original RLS Filter Expression in the Table Permission has been modified (Yellow) where an explicit User Principal Name in the dataset is added instead, to test (Green). The RLS code is executed inside of the ADDCOLUMNS iterator over a relevant part of the table. The checkmark indicates any row that evaluates to TRUE. The test demonstrates that the RLS is - for this UPN - working as expected, since <i>Gal Aehad</i> is the only user returning TRUE when their UPN is given.</figcaption>
+</figure>
+
+---
+
+### 5. Add a New Table Permission to a Role
+To add a new table permission:
+
+
+1. __Right-Click the Role:__ Select 'Add table permission...'
+
+<figure style="padding-top: 15px;">
+  <img class="noscale" src="~/images/data-security/data-security-table-permissions.png" alt="Data Security Create Role" style="width: 550px;"/>
+  <figcaption style="font-size: 12px; padding-top: 10px; padding-bottom: 15px; padding-left: 75px; padding-right: 75px; color:#00766e"><strong>Figure 6:</strong> In Tabular Editor, Table Permissions for RLS are visible under the role. New table permissions can be created by right-clicking a Role and selecting <i>'Add Table Partition...'</i></figcaption>
+</figure>
+
+2. __Select the Table and press 'OK':__ Select the table for which you want to create the permission.
+3. __Write the Filter Expression / RLS Table Permissions:__ Write the DAX for the filter expression. As above, you want to validate this filter expression (See __Figure 5__):
+  - Copy the Filter Expression to a new DAX Query window under an `EVALUATE` statement.
+  - Add it as the Expression of an `ADDCOLUMNS` statement iterating over Table, or part of the table.
+  - Execute it and observe the results.
+  - Replace `USERNAME()` or `USERPRINCIPALNAME()` in dynamic RLS with a known value from the Security Table.
+  - Re-run the DAX Query and validate that the results appear as expected. Repeat until satisfied.
+
+---
+
+### 6. Assign or Remove Users from a Role
+To assign users to a role:
+
+- _Power BI:_ This is done in the _Power BI Service_ from the Dataset Security settings.
```

Comment:
I would suggest avoiding screenshots of the Power BI service as much as possible because it seems to be going through such a dramatic UI change in the last 4 months. I'd agree in principle, but think we should try to leave PBI to PBI.